tests/cases/standalone/common/function/date.result

```diff
@@ -0,0 +1,164 @@
+SELECT date_add('2023-12-06 07:39:46.222726'::TIMESTAMP_MS, INTERVAL '5 day');
+
++-------------------------------------------------------------------------------------------+
+| date_add(Utf8("2023-12-06 07:39:46.222726"),IntervalMonthDayNano("92233720368547758080")) |
++-------------------------------------------------------------------------------------------+
+| 2023-12-11T07:39:46.222                                                                   |
++-------------------------------------------------------------------------------------------+
```

Comment:
Is '726' truncated expected? Rest LGTM

Reply:
I will change the literal to avoid the confusion.